recipes/simdutf/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "simdutf" %}
+{% set version = "4.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simdutf/simdutf/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e193c4b5912612147c266ea7ebe2108099f3bcab5f0dc34e6289b2a47495ae2f
+
+build:
+  number: 0
+
```

Shared libraries should come with a run_exports.
```suggestion
  run_exports:
    - {{ pin_subpackage('simdutf', max_pin='x') }}
```